jerry-libc/jerry-libc.c

```diff
@@ -158,41 +158,15 @@ CALL_PRAGMA (GCC diagnostic pop)
 int
 strcmp (const char *s1, const char *s2)
```

Comment:
BTW I'd prefer implementation that looks more like that:

```c
int strcmp(const char *str1, const char *str2)
{
    int s1;
    int s2;
    do {
        s1 = *str1++;
        s2 = *str2++;
        if (s1 == 0)
            break;
    } while (s1 == s2);
    return (s1 < s2) ? -1 : (s1 > s2);
}
```
Source: http://stackoverflow.com/a/29279634/230855